src/pytest/pytestTestRunner.ts

```diff
@@ -157,6 +157,7 @@ export class PytestTestRunner implements ITestRunner {
         const updatedPythonPath = [
             config.getCwd(),
             environment.PYTHONPATH,
```

Comment:
nit: Can you please rename `environment` here to something like `envFileEnvironment` so there won't be any confusion with the whole env?